maraca.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
I suggest you remove your .Rproj, because it contains your settings, and those will likely conflict with the settings of another user that may check out maraca.

In fact, Rstudio added a gitignore on .Rproj files, but this one slipped in anyway.